src/styles/_variables.scss

```diff
@@ -110,9 +114,19 @@ $h4-font-size-article:        $font-size-article * 1.33;
 $h5-font-size-article:        $font-size-article;
 $h6-font-size-article:        $font-size-article;
 
+$border-color:                $light;
+$table-border-color:          $border-color;
+$input-border-color:          $border-color;
+
+$input-padding-x:             1.2rem;
+$input-padding-y:             1rem;
+$input-padding-y-sm:          0.7rem;
+$input-padding-x-sm:          1.2rem;
+$input-border-radius:         3px;

```

Comment:
Do we need a specific border-radius for input, can't we use the border-radius variable?

Reply:
I will added global border-radius after I merge master to this branch